checker/src/org/checkerframework/checker/formatter/FormatterAnnotatedTypeFactory.java

```diff
@@ -147,5 +151,51 @@ public boolean isSubtype(AnnotationMirror rhs, AnnotationMirror lhs) {
 
             return super.isSubtype(rhs, lhs);
         }
+
+        @Override
+        public AnnotationMirror greatestLowerBound(AnnotationMirror anno1,
+                AnnotationMirror anno2) {
+            if (AnnotationUtils.areSameIgnoringValues(anno1, FORMAT) &&
+                AnnotationUtils.areSameIgnoringValues(anno2, FORMAT)) {
+                ConversionCategory[] anno1ArgTypes =
+                        treeUtil.formatAnnotationToCategories(anno1);
+                ConversionCategory[] anno2ArgTypes =
+                        treeUtil.formatAnnotationToCategories(anno2);
+
+                if (anno1ArgTypes == null && anno2ArgTypes == null) {
+                	return FORMAT;
+                }
+
+                if (anno1ArgTypes == null || anno2ArgTypes == null) {
+                    return FORMATBOTTOM; // TODO: Should this be InvalidFormat instead?
+                }
+
+
+                if (anno1ArgTypes.length != anno2ArgTypes.length) {
+                    return FORMATBOTTOM; // TODO: Should this be InvalidFormat instead? (Different situation from the if statement above)
```

Comment:
Yes, that is the case. And I agree it should be `@FormatBottom`, for the reason you stated.